packages/gridjs/src/view/table/td.tsx

```diff
@@ -10,11 +10,12 @@ import { PluginRenderer } from '../../plugin';
 
 export interface TDProps
   extends BaseProps,
-    JSX.HTMLAttributes<HTMLTableCellElement> {
```

Comment:
can you run prettifier on this please?

Reply:
this happens after prettifier, i double cheched in prettifier.net, same result.